commit 434c0b244341f586172f81a7476fe43ed1deea1c
Author: Phat Huynh <sleepyorion@gmail.com>
Date:   Thu Jul 6 07:27:06 2017 -0700

    suppress hashie warnings in older version of berks

diff --git a/lib/berkshelf.rb b/lib/berkshelf.rb
index f0f4cfc..f0bebfa 100644
--- a/lib/berkshelf.rb
+++ b/lib/berkshelf.rb
@@ -1,3 +1,9 @@
+# XXX: work around logger spam from hashie
+# https://github.com/intridea/hashie/issues/394
+require "hashie"
+require "hashie/logger"
+Hashie.logger = Logger.new(nil)
+
 require 'buff/extensions'
 require 'cleanroom'
 require 'digest/md5'
